app/request_handlers/handlers/lobby.js

```diff
@@ -88,4 +90,40 @@ const lobbyIsJoined = (req, res) => {
 	return res.send({"lobbyId" : -1});
 }
 
-module.exports = { lobbyJoin, lobbyJoinGame, lobbyLeave, lobbiesGet, lobbyIsJoined };
+const lobbyStats = (req, res) => {
+	const username = req.cookies.username;
+	const token = req.cookies.token;
+	const stats = {"wins": undefined, "losses": undefined};
+	
+	const pool = getPool();
+	if (CLIENTS_HANDLER.isUserLoggedin(token)) {
+		pool.query('SELECT wins, losses FROM userdata WHERE username=$1', [username]).then((result) => {
+			const data = result.rows;
+			if (data.length !== 0) {
+				stats.wins = data[0].wins;
+				stats.losses = data[0].losses;
+			}
+			return res.send(stats);
+		}).catch((error) => {
+			console.error('Error getting player stats:', error);
+			res.statusCode = 400;
+			return res.send(stats);
+		});
+	}
+	else {
+		return res.send(stats);
+	}
+}
+
+const getLeaderboard = (req, res) => {
```

Comment:
yooo a leaderboard?? It's cool that this is here, but there's no button on the page to display this, right? We can go to the route `.../lobby/leaderboard` though, which is fun enough